### ISSUE TITLE
Q: Is this project still alive?

### ISSUE THREAD

**Issue by JezhikLaas** [crozone/FormatWith#33]
Hello,
I really like this little helper - is it still alive? Help needed?

Best Regards, Uwe

**Comment by crozone** [comment]
Yes, but it has been neglected for a bit... I should really pick it back up and try to finish off v4.0.

The changes are on the release-4.0 branch, there's a few remaining issues to solve. I really want to get something similar to MS' stackalloc based ValueStringBuilder into the library and get it almost allocation free most of the time.

> Help needed?

PRs are always welcome :)
